plugins/ruby-foreman-remote-execution/debian/gem.list

```diff
@@ -1,3 +1,3 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_remote_execution-1.8.4.gem"
+GEMS="https://rubygems.org/downloads/foreman_remote_execution-2.0.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/foreman_remote_execution_core-1.2.0.gem"
```

Comment:
this should be 1.3.0, I guess? :)

Reply:
That's being done in here https://github.com/theforeman/foreman-packaging/pull/4166

Reply:
I see... :eyes: